config/routes.rb

```diff
@@ -1,6 +1,11 @@
-Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+# frozen_string_literal: true
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+Rails.application.routes.draw do
+  get "/users", to: "users#index", as: 'user'
+  get "/posts/new", to: "posts#new", as: 'posts_new'
+  post "posts", to: 'posts#create', as: 'posts_create'
+  get "/users/:id", to: "users#show", as: 'users'
+  get "/users/:user_id/posts", to: "posts#index", as: 'user_posts'
+  get "/users/:user_id/posts/:id", to: "posts#show", as: 'user_post'
+  post 'comments/:user_id/:post_id', to: 'comments#create', as: 'comments_create'
```

Comment:
- [ ] Please make sure that there a routes in the `config/routes.rb` file which corresponds with the new and create, actions in the comments controller.  You should have something  similar to: :arrow_down: 

```
   POST   /posts/:post_id/comments(.:format)     comments#create
   GET   /posts/:post_id/comments/new(.:format)     comments#new

```

- [ ] Please make sure that you have for like there is a route for creating route for posts similar to the one below :arrow_down:  

```
   POST   /posts/:post_id/likes(.:format)     likes#create
```